sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java

```diff
@@ -60,6 +61,7 @@
 import java.util.Arrays;
 import java.util.Date;
 import java.util.HashMap;
+import java.util.LinkedList;
```

Comment:
I'm not sure about using a LinkedList here - it will cause a bunch of allocation. We could use a fixed length array and pre-allocate all breadcrumbs. This would mean that adding a breadcrumb would not allocate any memory, causing less GC in the app.